testsuite/MDAnalysisTests/analysis/test_align.py

```diff
@@ -386,6 +386,11 @@ def test_alignto_partial_universe(self, universe, reference):
         assert_array_almost_equal(segB_bound.positions, segB_free.positions,
                                   decimal=3)
 
+    def test_alignto_sort(self, universe):
+        mobile = universe.atoms[:4]
+        ref = universe.atoms[[3, 2, 1, 0]]
+        assert align.alignto(mobile, ref, select='bynum 1-4') == (0.0, 0.0)
```

Comment:
I sometimes get a bit nervous on exact equality checks with floating point calculations, though probably not a big deal here. `np.assert_allclose()` may be slightly less risky, though I suppose one could argue the values really should be very close to identical.

Reply:
Fixed the code. I didn't take the margin of error of floating-point calculations seriously. 
https://github.com/MDAnalysis/mdanalysis/pull/3232/commits/109714eece66c6f231f8b8362ea4cce4d767bb6c